experimenter/experimenter/nimbus_ui_new/tests/test_views.py

```diff
@@ -988,3 +996,75 @@ def test_qa_edit_mode_post_invalid_form(self):
         self.experiment.refresh_from_db()
         self.assertNotEqual(self.experiment.qa_status, "INVALID_STATUS")
         self.assertEqual(self.experiment.qa_status, "NOT_SET")
+
+    def test_takeaways_card(self):
+        response = self.client.get(
+            reverse("nimbus-new-detail", kwargs={"slug": self.experiment.slug}),
+            **{settings.OPENIDC_EMAIL_HEADER: self.user_email},
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertContains(response, "Takeaways")
+        self.assertContains(response, "QBR learning")
+        self.assertContains(response, "Statistically significant DAU Gain")
+        self.assertContains(response, "This is a summary.")
+        self.assertContains(response, "0.5% gain in retention")
+        self.assertContains(response, "Graduate")
+        self.assertContains(response, "Rerun")
```

Comment:
Oh yeah, we should talk as a group about how we want to handle this.  Testing content in Django templates is definitely a debateable subject.  I've generally steered away from it, but let's talk about it as a group at the next engineering meeting.

Reply:
So looking at this in more detail @yashikakhurana , these are the labels that come from constants.  So rather than hard coding the text here, I'd probably structure these assertions to explicitly refer to the constants and assert that they're present in the page, rather than hard coding them as strings here.

Reply:
done, and also moved the forms into `nimbus_ui_new`